spec/lang/step/terminators.md

```diff
@@ -403,6 +486,63 @@ impl<M: Memory> Machine<M> {
 }
 ```
 
+## Catch unwinding
+
+```rust
+impl<M: Memory> Machine<M> {
+    /// Returns an invalid place (address 1) with the unit type.
+    fn unit_place() -> (Place<M>, Type) {
```

Comment:
This function is still used before it is declared. Please move it up by one section, to just before `eval_terminator(ResumeUnwind)`.